src/library/general_library.lua

```diff
@@ -526,5 +526,37 @@ function library.calc_script_name(include_extension)
     return retval
 end
 
+--[[
+% get_used_font_name
+
+Processes an input font name into Mac or Windows specific versions. This function is used by get_defualt_music_font().
+
+@ [standard_name] (string) The name of the font as returned by FCFontInfo:GetName().
+: (string) The OS-specific name of the font.
+]]
+function get_used_font_name(standard_name)
+    local font_name = standard_name
+    if string.find(os.tmpname(), "/") then
+        font_name = standard_name
+    elseif string.find(os.tmpname(), "\\") then
+        font_name = string.gsub(standard_name, "%s", "")
+    end
+    return font_name
+end
+
+--[[
+% get_default_music_font
+
+Fetches the default music font from document options and processes the name into a usable format.
+
+: (string) The name of the defalt music font.
+]]
+function get_default_music_font()
+    local fontinfo = finale.FCFontInfo()
+    if fontinfo:LoadFontPrefs(finale.FONTPREF_MUSIC) then
+        return getUsedFontName(fontinfo:GetName())
```

Comment:
The documentation for `GetName` specifically disrecommends its use, because it is not Unicode. Use `GetNameString` instead (and move the call to `calc_font_os_name`.)